src/generic/processing-notification/index.jsx

```diff
@@ -1,28 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { Badge, Icon } from '@openedx/paragon';
+import {
+  Badge, Button, Icon, Stack,
+} from '@openedx/paragon';
 import { Settings as IconSettings } from '@openedx/paragon/icons';
 import { capitalize } from 'lodash';
 
-const ProcessingNotification = ({ isShow, title }) => (
+const ProcessingNotification = ({ isShow, title, action }) => (
```

Comment:
> Do you have time to convert this component to a proper [Toast](https://paragon-openedx.netlify.app/components/toast/) ?

@pomegranited Took some time but it is done now.

Reply:
Please make the UX team aware of us. They originally asked us not to use Toast, because they wanted the same sort of animated notifications on the right-hand side of the screen that are used in the course experience. I do prefer the consistency of using Toast everywhere though.

Reply:
@bradenmacdonald We can move the Toast popup to the bottom-right (for dir=ltr) and make sure it's positioned over the sidebar? Then we get the benefits of the Toast component and UX get their positioning.

Reply:
@bradenmacdonald Huh, really? I don't see why not, but I don't have context on this discussion either. Will see what @navinkarkera can do.

Reply:
@pomegranited  See https://github.com/openedx/frontend-app-authoring/pull/1211 and [this comment](https://github.com/openedx/frontend-app-course-authoring/issues/1104#issuecomment-2286722658). As long as you can satisfy those concerns, please definitely convert this to a standard toast :)

Reply:
@bradenmacdonald @pomegranited Done. Moved standard toast over to right side and it has same animation as before and it automatically goes away after few seconds.

![image](https://github.com/user-attachments/assets/bb09bc3b-60d7-40cd-bfc6-42e47570bd1e)

Also, if no onClose handler is passed (which is the case in course outline and units), the close icon is hidden at which point it looks same as before.

![image](https://github.com/user-attachments/assets/4d66b4ac-6297-4ebd-bcc9-66ce56526019)